Source/WebCore/page/Quirks.cpp

```diff
@@ -672,6 +672,19 @@ bool Quirks::needsYouTubeOverflowScrollQuirk() const
 #endif
 }
 
+// FIXME: Remove after the site is fixed, <need>
+// youtube.com rdar://135886305
+bool Quirks::needsYouTubeDarkModeQuirk() const
+{
+    if (!needsQuirks())
+        return false;
+
+    if (!m_needsYouTubeDarkModeQuirk)
+        m_needsYouTubeDarkModeQuirk = m_document->url().host() == "www.youtube.com"_s;
```

Comment:
What about tv.youtube.com. studio.youtube.com, www.youtubekids.com etc?

Reply:
I don't think we have evidence that this issue exists on these.

Reply:
After some looking this issue also exists on studio.youtube.com. Made this for all YouTube.com subdomains.